Gemfile

```diff
@@ -1,21 +1,17 @@
 source 'https://rubygems.org'
 
-gem 'activesupport'
-gem 'timecop'
-gem 'webmock'
-gem 'cucumber', '~> 2.4'
-gem 'site_prism'
-gem 'rspec'
-gem 'rspec-its'
-gem 'spring'
-gem 'spring-commands-rspec'
-gem 'spring-commands-cucumber'
-gem 'pry'
-gem 'rubocop', require: false
-gem 'selenium-webdriver'
-gem 'poltergeist', '1.10.0'
-gem 'chromedriver-helper'
-gem 'capybara-screenshot'
-gem 'launchy'
-gem 'require_all'
-gem 'eyes_selenium'
+group :test do
+  gem 'capybara', '~> 2.15', '>= 2.15.1'
+  gem 'capybara-screenshot'
```

Comment:
Should we get into the habit of specifying versions (to 2 levels only to allow for minor updates) for every gem - even in the test environment.

Reply:
Done